codec-http/src/main/java/io/netty/handler/codec/http/websocketx/WebSocketClientProtocolHandshakeHandler.java

```diff
@@ -65,4 +94,44 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) throws Exception
             response.release();
         }
     }
+
+    public WebSocketClientProtocolHandshakeHandler handshakeTimeoutMillis(long handshakeTimeoutMillis) {
+        this.handshakeTimeoutMillis = handshakeTimeoutMillis;
+        return this;
+    }
+
+    public void applyHandshakeTimeout() {
```

Comment:
package-private as the class is package-private as well

Reply:
Done